### PR TITLE
[10.0][ADD] mrp_production_service

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
This module allows to create procurement orders from manufacturing orders when
a services is included in the Bill of Materials.

This allows users to include additional services that has to be procured as
part of the manufacturing process.

### CODE REVIEW

File: mrp_production_service/models/mrp_production.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models, _
```
Comment (bealdav) [inline mrp_production_service/models/mrp_production.py]: s/openerp/odoo

**Review by bealdav — APPROVED** [review]
Nice module for subcontracting operation for example

Really thanks

**Comment by HviorForgeFlow** [comment]
@jbeficent @lreficent can review?

**Review by LoisRForgeFlow — APPROVED** [review]
Tested locally :+1: 

Thanks

**Pull request merged**